sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>Nit: the title level of <code>Install the dependencies&nbsp;</code> does not seem correct.</p><p>This should not be under <code>Dataset</code> section  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>